### PR TITLE
Quest fixes Sept 2020

### PR DESCRIPTION
**Pull request by JayH2971**

### CODE REVIEW

File: Assets/StreamingAssets/Quests/R0C11Y28.txt [hunk 1]
```diff
@@ -215,9 +215,9 @@ _vamp_ is the =vamp_ I met over at _hookerhouse_. Nice enough, but spooky.
 _vamp_ is the name of the =vamp_ staying over at _hookerhouse_.
 
 Message:  1017
-_vamp_ is one of the %vcn, sent to %cn to eliminate the mad Slayer.
+_vamp_ is one of the %___bookstore_, sent to ___bookstore_ to eliminate the mad Slayer.
```
Comment (df-ferital) [inline Assets/StreamingAssets/Quests/R0C11Y28.txt:218]: Wrong search and replace there, %vcn should remain :)

File: Assets/StreamingAssets/Quests/R0C11Y28.txt [hunk 2]
```diff
@@ -215,9 +215,9 @@ _vamp_ is the =vamp_ I met over at _hookerhouse_. Nice enough, but spooky.
...
 <--->
-_vamp_ is a %vcn vampire here to stop the risk to %g3 security, the Slayer.
+_vamp_ is a %___bookstore_ vampire here to stop the risk to %g3 security, the Slayer.
```
Comment (df-ferital) [inline Assets/StreamingAssets/Quests/R0C11Y28.txt:220]: Same here.

File: Assets/StreamingAssets/Quests/R0C11Y28.txt [hunk 3]
```diff
@@ -313,7 +313,7 @@ Message:  1025
  trap if there ever was one.
 
 Message:  1026
-<ce>                 Which letter brought you to me, %pcn?
+<ce>                 Which letter brought you to me, %___bookstore_?
```
Comment (df-ferital) [inline Assets/StreamingAssets/Quests/R0C11Y28.txt:316]: Same here, this for %pcn.

File: Assets/StreamingAssets/Quests/R0C11Y28.txt [hunk 4]
```diff
@@ -348,14 +348,14 @@ Message:  1050
 <ce>                   By killing _rippername_, you have
 <ce>                 aided both %reg and myself. For this,
 <ce>                  I give you a _reward2_. Perhaps we
-<ce>                     may meet again one day, %pcn.
+<ce>                     may meet again one day, %___bookstore_.
```
Comment (df-ferital) [inline Assets/StreamingAssets/Quests/R0C11Y28.txt:351]: Same here.

File: Assets/StreamingAssets/Quests/R0C11Y28.txt [hunk 5]
```diff
@@ -364,12 +364,12 @@ Message:  1050
 --              %god occurs 3 times.
 --               %mn occurs 3 times.
 --              %pcf occurs 11 times.
---              %pcn occurs 2 times.
+--              %___bookstore_ occurs 2 times.
```
Comment (df-ferital) [inline Assets/StreamingAssets/Quests/R0C11Y28.txt:367]: Same here.

File: Assets/StreamingAssets/Quests/R0C11Y28.txt [hunk 6]
```diff
@@ -364,12 +364,12 @@ Message:  1050
...
 --             %qdat occurs 6 times.
 --               %ra occurs 1 time.
 --              %reg occurs 10 times.
 --               %rt occurs 1 time.
---              %vcn occurs 3 times.
+--              %___bookstore_ occurs 3 times.
```
Comment (df-ferital) [inline Assets/StreamingAssets/Quests/R0C11Y28.txt:372]: Same here, %vcn.

**Review by df-ferital — COMMENTED** [review]
Small mistakes :).

File: Assets/StreamingAssets/Quests/R0C11Y28.txt (hunk 6 above)
Comment (JayH2971) [reply Assets/StreamingAssets/Quests/R0C11Y28.txt:372]: Wow, that's terrible. Thanks a lot for proofreading that. I don't know how find and replace can go _that_ badly, but it did!

**Comment by Interkarma** [comment]
As always, I appreciate all your effort Jay. Thanks so much for fixing up these quests, and all the other stuff you do.

**Pull request merged**